### ISSUE TITLE
feature request: alarms x hours before after event

### ISSUE THREAD

**Issue by eaeaeaeaea** [forrestguice/SuntimesWidget#416]
Hi there,

I would really like to set up relative alarms
ie of the form:

'10 minutes before nautical sunset'
'5 hours after civil sunrise'

Thanks for your attention!

**Comment by forrestguice** [comment]
Hi. This is actually already possible (but it might not be obvious how to use it). The alarm UI still needs some work (planned for v0.13.0).

If you click on the word `at` above the alarm time, or select `Set Offset` from the alarm's overflow menu (`:`), it should allow setting a relative alarm.

**Comment by overchu** [comment]
I was looking for that feature and didn't discover it. Only when I searched through the issues I got here. But also after reading this I didn't find, trying everything I could think of.
Finnally I found which ~at~ you were reffering to. This is the most important feature for me in the app. Maybe you want to think about to make it more apparent in the UI. It would be a shame if other users who are looking for it will not find it.
This is such a great app! Thank you so much, I am very excited using it!
